src/components/policy/policy_external/src/policy_manager_impl.cc

```diff
@@ -1798,6 +1799,7 @@ bool PolicyManagerImpl::IsNeedToUpdateExternalConsentStatus(
   difference_v.resize(new_status_v.size() + existing_status_v.size());
 
   ItemV::iterator ci = difference_v.begin();
+  // cppcheck-suppress redundantAssignment
   ci = std::set_difference(new_status_v.begin(),
```

Comment:
@Ypostolov The `ci` variable could be initialized by the result of the `std::set_difference`.